custom_components/open_epaper_link/imagegen.py

```diff
@@ -175,12 +175,13 @@ def customimage(entity_id, service, hass):
             font_file = os.path.join(os.path.dirname(__file__), font)
             font = ImageFont.truetype(font_file, size)
             color = element.get('color', "black")
+            anchor = element.get('anchor', "lm")
```

Comment:
FYI this was a behavior change - the default anchor is `la` not `lm` - see https://pillow.readthedocs.io/en/stable/reference/ImageDraw.html#PIL.ImageDraw.ImageDraw.text

That messed up some of my tags